Highrachy/ballers-api
# #175634775 allow admin to approve vendor information

#### What does this PR do?
Allow admin to approve vendor information

#### Description of Task to be completed?
Allow admin to approve vendor information

#### How should this be manually tested?
`PUT /api/v1/user/vendor/verify`
`PUT /api/v1/user/vendor/verify/comment`
`PUT /api/v1/user/vendor/verify/step`

#### What are the relevant pivotal tracker stories?
#175634775
